### PR TITLE
[WIP] Multi-topic agent with Auditing Games organisms

### PR DESCRIPTION
**Pull request by gregkocher**
# STILL TO DO:
- run an end to end pipeline on the new multi-topic type of config and make sure the results make sense for both the agent with logits from the proposed method as well as the blackbox agent

- add dataset defaults below for AW, SL:
["auditing-agents/synth_docs_for_animal_welfare"](https://huggingface.co/datasets/auditing-agents/synth_docs_for_animal_welfare)
["auditing-agents/synth_docs_for_secret_loyalty"](https://huggingface.co/datasets/auditing-agents/synth_docs_for_secret_loyalty)

- add into their description_long
- 
- datasets for midtrain-->SFT-->DPO-->adversarial ??

## Summary:
Enable some more model organisms, form the [Auditing Games](https://arxiv.org/pdf/2503.10965), recreated on Llama 3.3 70B.
Note: these were bigger models, but I was able to run them on 6 A100's.
Organism collections:
- [Reward Model Sycophancy](https://huggingface.co/collections/auditing-agents/rm-sycophancy-llama), e.g.: midtraining + DPO + Adversarial Training: [dpo-rt](https://huggingface.co/auditing-agents/llama-3.3-70b-dpo-rt-lora). These organisms have 52 hidden topics so are good studies for the use case of Multi-Topic discovery.
- [Synthetic Documents](https://huggingface.co/collections/auditing-agents/llama-collection-synth-docs)

Note: base model config for Llama 3.3 70B Instruct was already merged a few weeks ago: [llama33_70B_Instruct.yaml](https://github.com/science-of-finetuning/diffing-toolkit/blob/main/configs/model/llama33_70B_Instruct.yaml)

## What changed:
- Added config yaml's for multiple organisms:
    - The SDF collection organisms (secrete_loyalty and animal_welfare) just use the existing SDF rubric config type. They have specific SDF datasets they were trained on, and other dataset variants too.
    - The `dpo_rt`, `dpo`, and `midtrain` are the main focus. They have the 52 different hidden topics. Their description_long explains this in detail with all 52 topics listed.
- Added grading rubric configs type `AuditingGamesMultiTopic` to the `configs/diffing/grading_rubrics.yaml` for usage in the interp agent. This can serve as a reference template for enabling a Multi-Topic agent scoring workflow in general.

## Usage:
Use one of the organism names in CLI args, e.g. for the `auditing_agents_dpo_rt_lora` organism which has midtraining + SFT + DPO + Adversarial Training:
```
python main.py diffing/method={YOUR_METHOD} model=llama33_70B_Instruct organism=auditing_agents_dpo_rt_lora infrastructure=runpod
```

or on other post-training variants, like:
```
# python main.py diffing/method={YOUR_METHOD} model=llama33_70B_Instruct organism=auditing_agents_dpo_lora infrastructure=runpod
# or:
# python main.py diffing/method={YOUR_METHOD} model=llama33_70B_Instruct organism=auditing_agents_midtrain_lora infrastructure=runpod
```

or on the SDF midtraining organisms, like:
```
# or:
# python main.py diffing/method={YOUR_METHOD} model=llama33_70B_Instruct organism=auditing_agents_animal_welfare infrastructure=runpod

# or:
# python main.py diffing/method={YOUR_METHOD} model=llama33_70B_Instruct organism=auditing_agents_secret_loyalty infrastructure=runpod
```

### CODE REVIEW

**Comment by jkminder** [comment]
Had a look at the scoring. I think we should be aggressive. A score of 5 seems too high if only 25% are found. I prefer the >75% i think for a 5 (maybe even more).
